### PR TITLE
improve(algolia): document addition of algolia facets

### PR DESCRIPTION
**Pull request by abailleulfc**
Document how to add a facet in algolia

https://deploy-preview-485--heuristic-almeida-1a1f35.netlify.app/docs/magento2/search-engine#requirements-1

### CODE REVIEW

File: docs/magento2/search-engine.mdx [hunk 1]
```diff
@@ -160,6 +160,13 @@ attribute and as a facet:
 You can then run the indexer so that the products are indexed in Algolia's
 index.
 
+:::note
+
+You can configure other custom attribute facets by perfoming the previous steps
+as for the `category_ids` attribute
```
Comment (dpobel) [inline docs/magento2/search-engine.mdx:166]: that's not super clear I think. I would phrase it like:

```suggestion
`category_ids` is the only required facets. Depending on your project, you can
configure any other attribute to be exposed as a facet by following the same
steps.
```

**Review by dpobel — COMMENTED** [review]
the same change also needs to be done for Magento1 in https://developers.front-commerce.com/docs/magento1/search-engine#requirements-1

**Comment by abailleulfc** [comment]
> the same change also needs to be done for Magento1 in https://developers.front-commerce.com/docs/magento1/search-engine#requirements-1

Applied to magento1

File: docs/magento2/search-engine.mdx [hunk 2]
```diff
@@ -160,6 +160,14 @@ attribute and as a facet:
 You can then run the indexer so that the products are indexed in Algolia's
 index.
 
+:::note
+
+`category_ids` is the only required facets. Depending on your project, you can
```
Comment (Kagaminara) [inline docs/magento2/search-engine.mdx:165]: @dpobel shouldn't `price` facet be required as well?

Comment (dpobel) [reply docs/magento2/search-engine.mdx:165]: it's not required as far as I can remember

**Review by dpobel — APPROVED** [review]

**Review by Kagaminara — APPROVED** [review]

**Pull request merged**